src/app/models/workspace.server.model.ts

```diff
@@ -75,6 +76,23 @@ const compileWorkspaceModel = (db: Mongoose): IWorkspaceModel => {
     admin: 1,
   })
 
+  WorkspaceSchema.virtual('count').get(function (this: IWorkspaceSchema) {
+    return this.formIds.length
+  })
+
+  WorkspaceSchema.statics.getWorkspaces = async function (
+    admin: IUserSchema['_id'],
+  ): Promise<WorkspaceDto[]> {
+    return this.find({ admin: admin }).sort('title').exec()
+  }
```

Comment:
comment: these are duplicate code from the form-workspaces-be/2 PR yea?

Reply:
Yep, it wasn't rebased yet when you saw it